### ISSUE TITLE
When training model by Asteriod...

### ISSUE THREAD

**Issue by Tigerwww** [mpariente/asteroid#348]
```
Epoch 0:   0%|          | 0/2 [00:00<?, ?it/s] Traceback (most recent call last):
  File "E:/PythonProjects/myDCCRN/test_tasnet.py", line 24, in <module>
    trainer.fit(system)
  File "B:\Anaconda\envs\py38cuda10\lib\site-packages\pytorch_lightning-0.9.1rc4-py3.8.egg\pytorch_lightning\trainer\trainer.py", line 302, in fit
    results = self.accelerator_backend.train()
  File "B:\Anaconda\envs\py38cuda10\lib\site-packages\pytorch_lightning-0.9.1rc4-py3.8.egg\pytorch_lightning\accelerators\cpu_backend.py", line 48, in train
    results = self.train_or_test()
  File "B:\Anaconda\envs\py38cuda10\lib\site-packages\pytorch_lightning-0.9.1rc4-py3.8.egg\pytorch_lightning\accelerators\base_backend.py", line 36, in train_or_test
    results = self.trainer.train()
  File "B:\Anaconda\envs\py38cuda10\lib\site-packages\pytorch_lightning-0.9.1rc4-py3.8.egg\pytorch_lightning\trainer\trainer.py", line 349, in train
    self.train_loop.run_training_epoch()
  File "B:\Anaconda\envs\py38cuda10\lib\site-packages\pytorch_lightning-0.9.1rc4-py3.8.egg\pytorch_lightning\trainer\training_loop.py", line 446, in run_training_epoch
    batch_output = self.run_training_batch(batch, batch_idx, dataloader_idx)
  File "B:\Anaconda\envs\py38cuda10\lib\site-packages\pytorch_lightning-0.9.1rc4-py3.8.egg\pytorch_lightning\trainer\training_loop.py", line 532, in run_training_batch
    response = self.trainer.call_hook('on_batch_start')
  File "B:\Anaconda\envs\py38cuda10\lib\site-packages\pytorch_lightning-0.9.1rc4-py3.8.egg\pytorch_lightning\trainer\trainer.py", line 639, in call_hook
    output = hook_fx(*args, **kwargs)
TypeError: on_batch_start() missing 1 required positional argument: 'batch'
Epoch 0:   0%|          | 0/2 [00:00<?, ?it/s]

Process finished with exit code 1

```
Hi, I'm trying to train the DCCRN on my own dataset but failed. I've checked my code and your scripts carefully but couldn't find the reason, and I tried your example of training ConvTasNet in [https://github.com/mpariente/asteroid/blob/master/notebooks/01_APIOverview.ipynb](url), however,  the same problem occurred. (Shown above)

Would you please tell me how to fix this? Very appreciate your help!

**Comment by jonashaag** [comment]
Hi! Can you please check the versions of PyTorch and PyTorch-Lightning that you are using? What version of Asteroid is this?

**Comment by Tigerwww** [comment]
> Hi! Can you please check the versions of PyTorch and PyTorch-Lightning that you are using? What version of Asteroid is this?


asteroid | 0.4.0rc0 |  
pytorch-lightning | 0.9.1rc4 |  
torch | 1.7.0+cu101 |

**Comment by jonashaag** [comment]
I wonder what's with the `0.9rc`, can you try to update to `0.9`?

**Comment by Tigerwww** [comment]
> I wonder what's with the `0.9rc`, can you try to update to `0.9`?

Yes! Problem solved! I re-install Pytorch-Lightning to `0.9` and everything is fine now. Very appreciate your help!
